### PR TITLE
German translation for v3_7_0

### PR DESCRIPTION
**Pull request by ann0see**
#343

### CODE REVIEW

File: wiki/de/de-Installation-for-Windows.md [hunk 1]
```diff
@@ -5,48 +5,68 @@ lang: "de"
...
+1. [Downloade Jamulus]({{ site.download_root_link }}{{ site.download_file_names.windows }}){: .button}\\
+**Alternativ-Download:** [SourceForge](https://sourceforge.net/projects/llcon/files/latest/download)
+1. **Installiere Jamulus**: Doppelklicke auf Installer, um die Installation zu starten. Wenn du eine Warnung von SmartScreen bekommst, klicke auf „Mehr Infos“ und „Trotzdem ausführen“, um Jamulus zu installieren. (Wenn du eine neue Version von Jamulus heruntergeladen hast und auch einer der Ersten warst, wird Jamulus noch nicht auf der SmartScreen-Whitelist sein. Du erhältst diese Meldung, da wir momentan nicht für ein Software Zertifikat für Jamulus zahlen.)
+1. **Starte Jamulus.** Jetzt kannst du Jamulus wie jedes andere Programm öffnen.
+1. **Richte deine Soundkarte ein**. Wenn du Jamulus installiert hast, musst du deine Audio-Hardware einrichten. Falls du ASIO4ALL nutzt, schau dir die [Einrichtung](#asio4all-einrichten) an und/oder das [Hardware-Setup](Hardware-Setup) an.
```
Comment (hoffie) [inline wiki/de/de-Installation-for-Windows.md]:
```suggestion
1. **Richte deine Soundkarte ein**. Wenn du Jamulus installiert hast, musst du deine Audio-Hardware einrichten. Falls du ASIO4ALL nutzt, schau dir die [Einrichtung](#asio4all-einrichten) und/oder das [Hardware-Setup](Hardware-Setup) an.
```

**Review by hoffie — CHANGES_REQUESTED** [review]
I've checked the diff between #343 and #347. One really minor duplicate word, after that, it's good to go? :)

**Comment by ann0see** [comment]
I think there's still a missing link to the onboarding page>connect to a server on the Windows page.

File: wiki/de/de-Installation-for-Windows.md [hunk 2]
```diff
@@ -5,48 +5,68 @@ lang: "de"
...
 
-Bevor du Jamulus startest solltest du:
+## ASIO4ALL einrichten
+*Du solltest ASIO4ALL nur dann nutzen, wenn deine Soundkarte keinen nativen, vom Hersteller der Soundkarte entwickelten ASIO-Treiber anbietet, da native ASIO-Treiber eine bessere Latenz haben könnten.*
 
-1. **Alle anderen Programme schließen** (besonders die, die (wie z.B. dein Browser/Media Player) auf deine Soundkarte zugreifen könnten), weil Jamulus exklusiven Zugriff auf die Soundkarte benötigt
-1. Sicherstellen, dass nur die **richtigen Ein-/Ausgänge** in ASIO4All **aktiv sind**. Alle anderen sollten ausgeschaltet sein. Wie das geht, siehst du hier:
+Wenn du ASIO4All nutzt, musst du möglicherweise ein bisschen herumprobieren, da jeder PC immer etwas anders ist.
+
+**Tipp:** Richte deine Soundkarte ein, wenn du mit einem Server verbunden bist Das erleichtert dir das Setup, da du dann mit deinem Instrument oder deiner Stimme testen kannst, ob alles richtig eingestellt ist; aber lies erst einmal weiter.
```
Comment (ann0see) [inline wiki/de/de-Installation-for-Windows.md]: @ann0see TODO: link onboarding!

**Comment by ann0see** [comment]
Should be done now. @gilgongo since you're changing the contributing page soon, should I still translate it?

**Pull request merged**